uint8 version       #0 for request, 1 for response
uint8 layer         # Layer data is from
uint16 position     # Key value offset
uint32 key_id       # KeyID of Configuration
string keyname      # Name of Configuration
uint64 data         # Configuration Data